Main.py

```diff
@@ -716,6 +716,7 @@ def create_playthrough(spoiler):
         # Otherwise, an item we collect could influence later item collection in the same sphere
         collected = list(search.iter_reachable_locations(item_locations))
         if not collected: break
+        random.shuffle(collected)
         # Gather the new entrances before collecting items.
         collection_spheres.append(collected)
         accessed_entrances = set(filter(search.spot_access, remaining_entrances))
```

Comment:
You should probably change this from `set(` to `list(` instead of making a new list later on.

Reply:
`accessed_entrances` needs to be a set for the `-=` 2 lines below this. For clarity, I'd rather convert to a list in the place where it needs to be a list rather than here.

Reply:
I guess it could be converted to a list in the `append` call.